otherfile/font_check.py

```diff
@@ -68,10 +68,12 @@ def print_result(flag_list, message_list):
     for font1 in Font_List:
         for font2 in Font_List:
             if Font_List.index(font1) < Font_List.index(font2):
-                if len(art.get_font_dic(font1)) == len(art.get_font_dic(font2)):
+                if len(
```

Comment:
What is this for?

Reply:
@sadrasabouri pep8 rule about max length of lines.

Reply:
Oh, Thanks.